.eslintrc.js

```diff
@@ -1,139 +1,115 @@
-/* eslint-disable quote-props */
 module.exports = {
-  'plugins': ['lodash-fp', 'simple-import-sort'],
-  'extends': 'react-app',
-  'settings': {
-    'import/resolver': {
-      'eslint-import-resolver-custom-alias': {
-        'alias': {
-          'src': './src',
-          'types': './types'
+  extends: ["airbnb-typescript-prettier"],
```

Comment:
looks like double-quotes in this file still need to be reverted to single-quotes, which should clean up the diff too.

Reply:
are there any more files you'd like me to do this on?(e.g. all other json files?)